lib/autorest/operations/index.d.ts

```diff
@@ -472,6 +475,26 @@ export interface Reports {
     cloneReport(collectionName: string, workspaceId: string, reportKey: string, requestParameters: models.CloneReportRequest, callback: ServiceCallback<models.Report>): void;
 
     /**
+     * @summary Deletes the specified report
```

Comment:
These changes are added to Swagger.json
As you see, C# code generated from swagger.json with delete report. Node didn't.
That's why I want to automate this process and I want one copy of the swagger to be there.